mod_reforged/hooks/skills/actives/fling_back_skill.nut

```diff
@@ -0,0 +1,7 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/fling_back_skill", function(q) {
+	// We delay the onFollow of this skill by another 100ms to account for faster combat speed mods
+	q.onFollow = @(__original) function( _tag )
+	{
+		::Time.scheduleEvent(::TimeUnit.Virtual, 100, __original, _tag);
```

Comment:
Add an `if` before this saying `if (::Time.getVirtualSpeed() > 2)` so we only add this delay when the speed is 2x or greater, because we know that up to a speed of 2 it works fine.